pkg/bindings/connection.go

```diff
@@ -307,17 +338,23 @@ func (h *APIResponse) IsServerError() bool {
 	return h.Response.StatusCode/100 == 5
 }
 
-func publicKey(path string) (ssh.AuthMethod, error) {
+func publicKey(path string, passphrase string) (ssh.AuthMethod, error) {
 	key, err := ioutil.ReadFile(path)
 	if err != nil {
 		return nil, err
 	}
 
 	signer, err := ssh.ParsePrivateKey(key)
 	if err != nil {
-		return nil, err
+		// TODO Should we prompt for passPhrase if Zero?
```

Comment:
It's perfectly fine, and even common practice, to have an empty passphrase

Reply:
@edsantiago How about prompting in the case that the key file requires it and it's not given?

Reply:
I would really expect that to be handled by the underlying ssh helper - it can't realistically be done anywhere else.